recipes/oslo.utils/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{%set name = "oslo.utils" %}
+{%set version = "3.16.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "109e018da9d95caba79c94935257d2335e5a77e65ebbff218cb9756e746630f1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - pbr >=1.6
```

Comment:
Not an issue. Just trying to understand the difference (and that it is intentional) is all. Thanks for explaining it.